components/Checkbox/Checkbox.jsx

```diff
@@ -108,31 +108,40 @@ const HiddenCheckbox = styled.input.attrs({
 
 HiddenCheckbox.displayName = 'HiddenCheckbox';
 
-const Checkbox = ({ label, error, id, ...rest }) => (
+const Checkbox = ({ children, error, id, label, value, ...rest }) => (
   <Wrapper>
     <CheckboxWrapper>
-      <HiddenCheckbox id={id} error={error} {...rest} />
+      <HiddenCheckbox id={id} error={error} value={value} {...rest} />
       <CheckIcon />
-      <CheckboxLabel htmlFor={id}>{label}</CheckboxLabel>
+      <CheckboxLabel htmlFor={id}>{children || label || value}</CheckboxLabel>
     </CheckboxWrapper>
-    {error && <ErrorMessage>{error}</ErrorMessage>}
+    {error && typeof error === 'string' && <ErrorMessage>{error}</ErrorMessage>}
   </Wrapper>
 );
 
 Checkbox.defaultProps = {
-  checked: undefined,
+  checked: false,
+  children: '',
   disabled: false,
   error: '',
   id: '',
   label: '',
+  onChange: () => {},
+  value: '',
 };
 
 Checkbox.propTypes = {
+  children: PropTypes.string,
   checked: PropTypes.bool,
   disabled: PropTypes.bool,
-  error: PropTypes.string,
+  error: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
```

Comment:
Because when checkboxes are inside a group, they share the same error message, below them:

<img width="149" alt="Screen Shot 2019-04-16 at 09 36 50" src="https://user-images.githubusercontent.com/755101/56209976-3759c100-602b-11e9-9b55-6b0e217c3ab6.png">

So, in this case, it's only required to mark them with an error style, not a message string.

Reply:
So, only `HiddenCheckbox` must have `error` prop as a boolean value and not the `Checkbox`

Reply:
`Checkbox` needs to evaluate [here](HiddenCheckbox) if it is an string, to check if the an error message must be shown.

Reply:
If it is a string, there is no need for type checking.
Just ```{error && <ErrorMessage>{error}</ErrorMessage>}``` would be enough.